Makefile

```diff
@@ -64,10 +64,17 @@ endif
 
 all: manager
 
+.PHONY: checkall
+check: test lint-golint
```

Comment:
Since the current target flow is `all` > `manager` > `test`, I suggest removing the `test` target from `check` and replace `lint-golint` with `check` in L71.

Reply:
"check" and "test" should be the same. "check" is for people who have muscle memory from auto tools :)

Reply:
> "check" and "test" should be the same.

I disagree. "test" should run your tests, "check" should run your tests and verify other checks against the branch.

Reply:
@danehans I took out the linting from `test` which should keep `test` the way it was before. To run linters then you need to run `check` which will do both.